core/MetaDataBroker/README.md

```diff
@@ -3,3 +3,15 @@
 Shortcut: `IDS-MDB`
```

Comment:
from now on are classes, which dont belong to the DAPS topic (which should be the pull-request)